### PR TITLE
JBPM-8878 : Error appears after clicking New Instance in MultipleSubForm Field properties modal window Fixed

### PR DESCRIPTION
**Pull request by akumar074**
- Added the check for standalone class name to generate TypeKind in Field definitions.
 - Removed type check in Providers.
 - Added Test for getFieldTypeInfo method.

### CODE REVIEW

**Comment by akumar074** [comment]
jenkins execute full downstream build

**Comment by jstastny-cz** [comment]
The https://issues.jboss.org/browse/RHPAM-1027 issue is not acked even by Engineering. Why is there a PR already?

**Comment by pefernan** [comment]
@jstastny-cz acked!

File: kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/input/impl/StringMultipleInputFieldDefinitionTest.java [hunk 1]
```diff
@@ -33,4 +38,13 @@ protected StringMultipleInputFieldDefinition getFullFieldDefinition() {
...
         return fieldDefinition;
     }
+
+    @Test
+    public void testGetFieldType() {
+        StringMultipleInputFieldDefinition fieldDefinition = new StringMultipleInputFieldDefinition();
+        fieldDefinition.setStandaloneClassName(String.class.getName());
+        assertEquals(fieldDefinition.getFieldTypeInfo().getType(), TypeKind.BASE);
+        fieldDefinition.setStandaloneClassName(Object.class.getName());
+        assertNotEquals(fieldDefinition.getFieldTypeInfo().getType(), TypeKind.BASE);
```
Comment (jstastny-cz) [inline kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/input/impl/StringMultipleInputFieldDefinitionTest.java]: better would be to check if is equal to TypeKind.OBJECT, wouldn't it?

File: kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/selector/impl/StringMultipleSelectorieldDefinitionTest.java [hunk 2]
```diff
@@ -45,4 +51,13 @@ protected StringMultipleSelectorFieldDefinition getFullFieldDefinition() {
...
         return fieldDefinition;
     }
+
+    @Test
+    public void testGetFieldType() {
+        StringMultipleInputFieldDefinition fieldDefinition = new StringMultipleInputFieldDefinition();
+        fieldDefinition.setStandaloneClassName(String.class.getName());
+        assertEquals(fieldDefinition.getFieldTypeInfo().getType(), TypeKind.BASE);
+        fieldDefinition.setStandaloneClassName(Object.class.getName());
+        assertNotEquals(fieldDefinition.getFieldTypeInfo().getType(), TypeKind.BASE);
```
Comment (jstastny-cz) [inline kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/selector/impl/StringMultipleSelectorieldDefinitionTest.java]: again, check equality to TypeKind.OBJECT

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Comment by akumar074** [comment]
jenkins execute full downstream build

File: kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/input/impl/StringMultipleInputFieldDefinitionTest.java (hunk 1 above)
Comment (akumar074) [reply kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/input/impl/StringMultipleInputFieldDefinitionTest.java]: done

File: kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/selector/impl/StringMultipleSelectorieldDefinitionTest.java (hunk 2 above)
Comment (akumar074) [reply kie-wb-common-forms/kie-wb-common-forms-core/kie-wb-common-forms-fields/src/test/java/org/kie/workbench/common/forms/fields/shared/fieldTypes/basic/lists/selector/impl/StringMultipleSelectorieldDefinitionTest.java]: done

**Comment by akumar074** [comment]
jenkins execute full downstream build please

**Pull request merged**